website/docs/project_settings/settings_project_global.md

```diff
@@ -263,10 +263,25 @@ Template may look like `"{family}{Task}{Variant}"`.
 Some creators may have other keys as their context may require more information or more specific values. Make sure you've read documentation of host you're using.
 
 
-## Workfiles
+### Publish
+
+#### Custom Staging Dir Profiles
+With this feature, users can specify a transient data folder path based on presets, which can be used during the creation and publishing stages.
+
+![global_tools_custom_staging_dir](assets/global_tools_custom_staging_dir.png)
+
+Staging dirs are used as a destination for intermediate files (as renders) before they are renamed and copied to proper location during integration phase. They could be created completely dynamically in temp folders, or for some DCCs in `work` area.
+(Example could be Nuke where artist might want to temporarily render pictures into `work` area to check them before they get published.)
```

Comment:
I don't see a reason why this needs to be between brackets in the documentation. It's an example use case which is what the documentation should explain if it's unclear without.

Additionally I have an issue with this example - because how does it allow them to "check the render" before they get published? The publisher still continues by default to actually publish and integrate the files, no? Unless maybe this is rendered on a farm and the submit publish job is suspended by default. To me this explanation in the docs hints a bit too much that it'll give me control to "check" before it integrates into the database which will not be the case for a lot of the publishes.

Reply:
Yeah, I am maybe using too much parentheses, even my czech language teacher was telling me that. That's Terry Pratchett fault ;)

Reply:
Nuke can have that "Use existing frames" in write node, so I think Jezza was describing that as a use case, but maybe I misunderstood.

Reply:
> Nuke can have that "Use existing frames" in write node, so I think Jezza was describing that as a use case, but maybe I misunderstood.

Ah, yes. That's an additional issue that could be problematic with this PR.

Are we sure this feature is worth the amount of 'likely won't work with X or Y' issues we're already assuming are there. 😄

Reply:
I think it is customer request and they will probably use it in pretty limited way, here we are trying to solve this for all the imaginary others and all the possible use cases, but deciding what feature and what uses cases it should cover is beyond my pay grade, I am afraid ;).